README.md

```diff
@@ -166,6 +166,205 @@ This solution collects anonymous operational metrics to help AWS improve the
 quality of features of the solution. For more information, including how to disable
 this capability, please see the [implementation guide](https://docs.aws.amazon.com/solutions/latest/amazon-marketing-cloud-uploader-from-aws/collection-of-operational-metrics.html).
 
+---
+# Advanced Usage
+
+## Integration with solution for file upload automation
+
+This solution's `start_amc_transformation` API endpoint can be integrated with other ETL pipelines to provide with the built-in normalization and timeseries partitioning before uploading data to Amazon Marketing Cloud (AMC).
+
+The purpose of the endpoint is to normalize and hash clear-text PII and partition time series data sets for AMC. Timestamp columns must be formatted before using the endpoint according to ISO 8601.
+
+The `start_amc_transformation` API endpoint starts an AWS Glue Job with the arguments passed into it. The following is a summary of the arguments included in a request to the endpoint and expected outputs of the API:
+
+### Inputs
+
+- `countryCode`: string
+	- The country code to be used for different country-specific normalizations to apply to all rows in the dataset
```

Comment:
should we add something about the countryCode options? maybe something like "the list of available countries can be found in the Implementation Guide" since it will change over time but the IG will serve as the source of truth (outside of the code!)